### PR TITLE
docs: fix psql command and pstgres data volume in quickstarts

### PR DESCRIPTION
**Pull request by Askir**

### CODE REVIEW

File: docs/vectorizer-quick-start.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Now we can create and run a vectorizer. A vectorizer is a pgai concept, it proce
 
 1. **Connect to the database in your local developer environment**
 
-   - Docker: `docker compose exec -it pgai psql`
+   - Docker: `docker exec -it pgai-db-1 psql -U postgres`
```
Comment (JamesGuthrie) [inline docs/vectorizer-quick-start.md:56]: Why did you change this back?

Comment (Askir) [reply docs/vectorizer-quick-start.md:56]: The original command was wrong it would need to be `-it db` not `-it pgai`. Hervè called this out.
The other two quickstarts simply use `docker exec` instead of `docker compose exec` though, so I just aligned them to that.

I don't really have any preference.

Comment (JamesGuthrie) [reply docs/vectorizer-quick-start.md:56]: Oof, ya you're right. I think that might have been me 🙈

Comment (JamesGuthrie) [reply docs/vectorizer-quick-start.md:56]: I have a slight preference for `docker compose` because you can just use the service name (without the `pgai-` and `-` prefixes and suffixes).

**Review by JamesGuthrie — APPROVED** [review]

File: docs/vectorizer-quick-start.md (hunk 1 above)
Comment (Askir) [reply docs/vectorizer-quick-start.md]: alright fair, changed it :D
